backend/src/cache/server/BUILD.bazel

```diff
@@ -0,0 +1,47 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "admission.go",
+        "client_manager_fake.go",
+        "mutation.go",
+        "watcher.go",
+    ],
+    importpath = "github.com/kubeflow/pipelines/backend/src/cache/server",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//backend/src/cache/client:go_default_library",
+        "//backend/src/cache/model:go_default_library",
+        "//backend/src/cache/storage:go_default_library",
+        "//backend/src/common/util:go_default_library",
+        "@com_github_golang_glog//:go_default_library",
+        "@com_github_peterhellberg_duration//:go_default_library",
```

Comment:
wondering whether we can change codes to use golang prebuilt Duration to avoid introducing 3rd lib.

Reply:
https://golang.org/pkg/time/#Duration.Hours

Reply:
https://stackoverflow.com/questions/28125963/golang-parse-time-duration

Reply:
never mind, that 3P lib is simple and small.

Reply:
Hi Renmin, this 3P is used for parse ISO8601 duration(eg. P30DT1H) to golang time.Duration which is not built in golang's Duration pkg.